process/block/shardblock.go

```diff
@@ -680,6 +721,12 @@ func (sp *shardProcessor) CommitBlock(
 		return err
 	}
 
+	err = sp.checkEpochCorrectnessCrossChain(chainHandler)
+	sp.epochStartTrigger.SetProcessed(header)
```

Comment:
It is ok to set processed this header even if there is an err return by checkEpochCorrectnessCrossChain ?

Reply:
put set only if not error.